scanpy/plotting/_tools/scatterplots.py

```diff
@@ -44,6 +45,10 @@ def plot_scatter(
     legend_fontweight: str = 'bold',
     legend_loc: str = 'right margin',
     legend_fontoutline: Optional[int] = None,
+    vmax: Union[float, str, callable, Sequence, None] = None,
+    vmin: Union[float, str, callable, Sequence, None] = None,
+    add_contour: Optional[bool] = False,
+    contour_config: Optional[dict] = {'color': ('black', 'white'), 'edge_width': (0.3, 0.05)},
```

Comment:
Why not `contour_edgewidth` and `contour_color`? This way people can’t misspell their dict keys.